src/screens/Home/Home.module.css

```diff
@@ -60,7 +60,7 @@
 }
 
 .first {
```

Comment:
can't all of these items have another class adding just background that repeats here?

Reply:
I changed background of all slides just for now and left them in case there will be different background in the future.